wzdx/tools/cdot_geospatial_api.py

```diff
@@ -0,0 +1,272 @@
+import json
+import logging
+import time
+
+import requests
+import os
+import sys
+
+from ..tools import geospatial_tools, path_history_compression
+
+
+class GeospatialApi:
+    def __init__(
+        self,
+        getCachedRequest=lambda x: None,
+        setCachedRequest=lambda x, y: None,
+        BASE_URL=os.getenv(
+            "CDOT_GEOSPATIAL_API_BASE_URL",
+            "https://dtdapps.colorado.gov/server/rest/services/LRS/Routes_withDEC/MapServer/exts/CdotLrsAccessRounded",
+        ),
+    ):
+        self.getCachedRequest = getCachedRequest
+        self.setCachedRequest = setCachedRequest
+        self.BASE_URL = BASE_URL
+        self.ROUTE_BETWEEN_MEASURES_API = "RouteBetweenMeasures"
+        self.GET_ROUTE_AND_MEASURE_API = "MeasureAtPoint"
+        self.GET_POINT_AT_MEASURE_API = "PointAtMeasure"
+        self.GET_ROUTES_API = "ROUTES"
+        self.GET_ROUTE_API = "ROUTE"
+        self.SR = "4326"
+        self.responseTimes = []
+
+    def _make_web_request(self, url: str, timeout):
+        resp = requests.get(url, timeout=timeout).content.decode("utf-8")
+        return resp
+
+    def get_routes_list(self):
+        parameters = []
+        parameters.append(f"f=pjson")
+
+        url = f"{self.BASE_URL}/{self.GET_ROUTES_API}?{'&'.join(parameters)}"
+        logging.debug(url)
```

Comment:
Looks like we have a couple of these logging statements with just the URL and then a comment with the logged data? If we want to log the URL in debug mode, should we indicate where it's being logged from, or some other identifying info, and remove the comment with the log output?

Reply:
Updated in https://github.com/CDOT-CV/Work_Zone/pull/55